src/backend/resourcemanager/include/resourcepool.h

```diff
@@ -715,6 +715,7 @@ SimpStringPtr build_segment_status_description(SegStat segstat);
 #define EPSILON 1e-7
 #define IS_DOUBLE_ZERO(d)       (fabs(d) < EPSILON)
 #define IS_DOUBLE_EQ(x, y)      ((fabs((x) - (y))) <= (EPSILON))
+#define IS_DOUBLE_EQGREATER(x, y)  (((x) + EPSILON) >= (y))
```

Comment:
I prefer to using IS_DOUBLE_EGT(equal greater than) instead of IS_DOUBLE_EQGREATER,
a little suggestion.